test/destest.c

```diff
@@ -697,6 +698,79 @@ static int test_des_quad_cksum(void)
         return 0;
     return 1;
 }
+
+/*
+ * Test TDES based key wrapping.
+ * The wrapping process uses a randomly generated IV so it is difficult to
```

Comment:
There are some KATS here... (A single decrypt test would be better than nothing) that could be added to evp_test maybe?
https://csrc.nist.gov/CSRC/media/Projects/Cryptographic-Algorithm-Validation-Program/documents/mac/kwtestvectors.zip

Reply:
Just nothing that exactly _zero_ of NIST's tests cover the case that was reported....

Reply:
These are tests for a different version of the algorithm :(